IFC4x3/Properties/w/WorkingState_2qD57tYLb86QIkv8vmjv3f/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the working state of Anemometrograph.
```

Comment:
Generalize?

Reply:
Generalized in the description: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/af8eaa192a6dc6d7f897e8651d218b992cf09a09